### PR TITLE
add question to test with official server

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
#893

### CODE REVIEW

File: issue_template.md [hunk 1]
```diff
@@ -9,6 +9,12 @@ Tell us what happens instead
 2. 
 3. 
 
+### Can this problem be reproduced with the official owncloud server?
+### Please log in with:
```
Comment (rperezb) [inline issue_template.md]: @tobiasKaminsky i'd remove the bold letters here

**Comment by tobiasKaminsky** [comment]
@rperezb updated the text

**Comment by rperezb** [comment]
@tobiasKaminsky thx!

this is on the radar to be merged soooon

**Comment by davivel** [comment]
This is OK and doesn't need QA.

@tobiasKaminsky , could you rebase on master before merging? Thanks

**Comment by davivel** [comment]
Mmm, owncloud-robot updated again... I'll rebase, @tobiasKaminsky ,  if you don't mind.

**Comment by davivel** [comment]
:+1:, thanks :)

**Pull request merged**